tasks/copy_to_glacier/README.md

```diff
@@ -353,7 +353,7 @@ FUNCTIONS
                     "ingestTime" (str): Date and time the file was originally ingested into ORCA.
                     "etag" (str): etag of the file object in the AWS S3 Glacier bucket.
     
-    get_default_glacier_bucket_name(config: Dict[str, Any]) -> str
+    get_destination_bucket_name(config: Dict[str, Any]) -> str
```

Comment:
Made this name a bit more general since we are now using storage classes other than `GLACIER`

Reply:
looks good to me.